.github/workflows/frontend_runtime_application_manual_e2e_run.yml

```diff
@@ -0,0 +1,186 @@
+name: Default workflow for a runtime application when E2E tests get triggred by a comment on the PR
+
+on:
+  workflow_call:
+    inputs:
+      fallback_runner:
+        description: "If true will leverage ubuntu-latest, otherwise will fall back to the J1 in-house runner"
+        default: false
+        type: boolean
+      e2e_filter_tags:
+        description: "Tests will be filtered based on the tags defined here"
+        type: string  
+      e2e_containers:
+        description: "The number of tests that you want Cypress to run in parallel (ex. 1, 2, 3, ...)"
+        type: string
+        default: '["1"]'
```

Comment:
The default value doesn't match the example provided in the description

Reply:
In terms of the very first example from `web-home`, that's the old flow and is also based on the [e2e_container input](https://github.com/JupiterOne/web-home/blob/cca0441a0ab3081649fa0c90bd49c02bd3d4b6fb/.github/workflows/pull_request.yml#L12) shown here for that repo. Each repo gets to decide how many containers they need for running their tests to optimize their test runs.

In terms of the other examples referenced above, these are from tests being run in specific repos, [web-navbar](https://github.com/JupiterOne/web-navbar/pull/380) for the `Application utility flow` and [web-query-results](https://github.com/JupiterOne/web-query-results/pull/219) for the `Utility runtime flow`.

The value shown here is for the global workflow and as demonstrated by remarks above, can be overridden by the consumer.

Reply:
Sorry, should've been more clear. The description says "ex. 1,2,3", ie: a string, but the default value is a string that is an array of strings `["1"]`. So does the user need to provide an array of strings, or just a string?

Reply:
Okay [updated the comment](https://github.com/JupiterOne/.github/blob/f5a7bf69c1f1ff5ed94a151ec8d554502ff7ac27/.github/workflows/frontend_runtime_application_manual_e2e_run.yml#L14) to the following:

<img width="798" alt="Screenshot 2023-10-27 at 4 42 07 PM" src="https://github.com/JupiterOne/.github/assets/1964654/6ffb7122-07b6-48af-b9f9-282de2396acd">